pvnet/models/model_cards/pvnet_india_model_card_template.md

```diff
@@ -0,0 +1,51 @@
+---
+{{ card_data }}
+---
+
+
+
+
+
+
+# PVNet India
+
+## Model Description
+
+<!-- Provide a longer summary of what this model is/does. -->
+This model class uses numerical weather prediction and recent wind power output to forecast the PV power in North West India over the next 48 hours. More information can be found in the model repo [1] and experimental notes [here](https://github.com/openclimatefix/PVNet/tree/main/experiments/india).
```

Comment:
I'm nitpicking into the sunset here, but I think technically only solar power is PV power. Unless I'm misunderstanding what it says of course!